### PR TITLE
Add documentation for Overseerr integration

### PR DESCRIPTION
**Pull request by l-holter**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [X] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [X] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/111960
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/5257
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by klaasnicolaas** [comment]
Missing a link to the PR in the core repository.

./Klaas

File: source/_integrations/overseerr.markdown [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ha_domain: overseerr
+ha_platforms:
+  - sensor
+ha_codeowners:
+  - '@vehjelmtvedt'
+  - '@l-holter'
+ha_integration_type: service
+---
+
+This Overseerr integration pulls data from a given [Overseerr](https://overseerr.dev/) instance.
```
Comment (klaasnicolaas) [inline source/_integrations/overseerr.markdown]:
```suggestion
This **Overseerr** {% term integration %} pulls data from a given [Overseerr](https://overseerr.dev/) instance.
```

File: source/_integrations/overseerr.markdown [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+Your API key can be found in Settings > General > API Key in the Overseerr Web UI.
```
Comment (klaasnicolaas) [inline source/_integrations/overseerr.markdown]:
```suggestion
Your API key can be found in **Settings** > **General** > **API Key** in the Overseerr Web UI.
```

File: source/_integrations/overseerr.markdown [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+Make sure to add /api/v1 to your URL in the configuration.
```
Comment (klaasnicolaas) [inline source/_integrations/overseerr.markdown]:
```suggestion
Make sure to add `/api/v1` to your URL in the configuration.
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/overseerr.markdown (hunk 3 above)
Comment (l-holter) [reply source/_integrations/overseerr.markdown]: Thank you, I have applied the changes:)

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**